bundles/org.openhab.binding.dirigera/src/main/java/org/openhab/binding/dirigera/internal/handler/sensor/MotionSensorHandler.java

```diff
@@ -0,0 +1,284 @@
+                int seconds = -1;
+                if (command instanceof DecimalType decimal) {
+                    seconds = decimal.intValue();
+                } else if (command instanceof QuantityType<?> quantity) {
+                    QuantityType<?> secondsQunatity = quantity.toUnit(Units.SECOND);
+                    if (secondsQunatity != null) {
+                        seconds = secondsQunatity.intValue();
+                    }
+                }
+                if (seconds > 0) {
+                    String updateData = String
+                            .format(gateway().model().getTemplate(Model.TEMPLATE_SENSOR_DURATION_UPDATE), seconds);
+                    sendPatch(new JSONObject(updateData));
+                }
+                break;
+            case CHANNEL_SCHEDULE:
+                if (command instanceof DecimalType decimal) {
+                    switch (decimal.intValue()) {
+                        case 0:
+                            gateway().api().sendPatch(config.id,
+                                    new JSONObject(gateway().model().getTemplate(Model.TEMPLATE_SENSOR_ALWQAYS_ON)));
+                            break;
+                        case 1:
+                            gateway().api().sendPatch(config.id,
+                                    new JSONObject(gateway().model().getTemplate(Model.TEMPLATE_SENSOR_FOLLOW_SUN)));
+                            break;
+                        case 2:
+                            String template = gateway().model().getTemplate(Model.TEMPLATE_SENSOR_SCHEDULE_ON);
+                            gateway().api().sendPatch(config.id,
+                                    new JSONObject(String.format(template, startTime, endTime)));
+                            break;
+                    }
+                }
+                break;
+            case CHANNEL_SCHEDULE_START:
+                String startSchedule = gateway().model().getTemplate(Model.TEMPLATE_SENSOR_SCHEDULE_ON);
+                if (command instanceof StringType string) {
+                    // take string as it is, no consistency check
+                    startTime = string.toFullString();
+                } else if (command instanceof DateTimeType dateTime) {
+                    startTime = dateTime.getZonedDateTime().format(timeFormat);
+                }
+                gateway().api().sendPatch(config.id, new JSONObject(String.format(startSchedule, startTime, endTime)));
+                break;
+            case CHANNEL_SCHEDULE_END:
+                String endSchedule = gateway().model().getTemplate(Model.TEMPLATE_SENSOR_SCHEDULE_ON);
+                if (command instanceof StringType string) {
+                    endTime = string.toFullString();
+                    // take string as it is, no consistency check
+                } else if (command instanceof DateTimeType dateTime) {
+                    endTime = dateTime.getZonedDateTime().format(timeFormat);
```

Comment:
This method is deprecated. 
More details here including links to core: https://github.com/openhab/openhab-addons/pull/17725

Reply:
Changed to `format(pattern,ZonedId.systemDefault())`